db/db.pl

```diff
@@ -67,7 +67,7 @@
 # 71 - user.roles, user.cont3xt
 # 72 - save es query in history, hunt description
 # 73 - hunt roles
-# 74 - shortcut sharing with users/roles
+# 74 - shortcut sharing with users/roles, notifiers index
```

Comment:
This will need to be 75 because those that already installed 74 (like us) won't get notifiersCreate/Move called.